src/coreclr/src/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -319,6 +326,15 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
                 GetEmitter()->emitIns_R_R(ins, emitSize, op2Reg, op1Reg, opt);
                 break;
 
+            // mvni won't take 8B/16B, so hard-code the size and reinterpret the bits. The behaviour is the same.
+            case NI_Vector64_get_Zero:
+            case NI_Vector64_get_AllBitsSet:
+                GetEmitter()->emitIns_R_I(ins, emitSize, targetReg, ival, INS_OPTS_2S);
```

Comment:
Can you please hard-code `ival` value here and remove call to `HWIntrinsicInfo::lookupIval` - I am removing `HWIntrinsicInfo::lookupIval` and the `ival` column from hwintrinsiclistarm64.h on Arm64 in #35030